Makefile

```
@@ -503,7 +503,7 @@ check-release-tag: ## Check if the release tag is set
 
 .PHONY: create-gh-release
 create-gh-release:$(GH) ## Create release on Github
-	$(GH) release create $(VERSION) -d -F $(RELEASE_DIR)/CHANGELOG.md -t $(VERSION) -R $(GH_REPO)
+	$(GH) release create $(VERSION) --draft --notes-file $(RELEASE_DIR)/CHANGELOG.md --title $(VERSION) --repo $(GH_REPO)
```

```suggestion
	$(GH) release create $(VERSION) --notes-file $(RELEASE_DIR)/CHANGELOG.md --title $(VERSION) --repo $(GH_REPO)
```

Move the draft option into `.goreleaser.yaml`; see comment on that file.